CMakeLists.txt

```
@@ -295,13 +261,12 @@ SET(CPACK_COMPONENT_HEADERS_DISABLED TRUE)
 SET(CPACK_COMPONENT_HEADERS_DEPENDS libraries)
 SET(CPACK_COMPONENT_EXAMPLES_DEPENDS libraries)
 
+# Binary Package
+SET(CPACK_GENERATOR "ZIP" CACHE STRING "Default binary package generator")
```

I am quite unhappy with this choice (and the choice made before). On Linux it is quite unusual to use zip.

```suggestion
if(WIN32)
    set(CPACK_GENERATOR ZIP WIX)
elseif(APPLE)
    set(CPACK_GENERATOR TGZ productbuild)
elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
    set(CPACK_GENERATOR TGZ RPM DEB)

    # Automatically include runtime dependencies (libc, libstdc++) fro Debian
    # (RPM includes these dependencies automagically)
    # YMMV, therefore commented
    # set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS "TRUE")
else()
    set(CPACK_GENERATOR TGZ)
endif()
```

---

Thank you for your suggestion. :)